metadeploy/api/models.py

```diff
@@ -847,6 +878,56 @@ def run(self, ctx, plan, steps, org):
         flow_coordinator.run(org)
 
 
+class ScratchOrg(HashIdMixin, models.Model):
+    Status = Choices("started", "complete", "failed", "canceled")
+
+    plan = models.ForeignKey(Plan, on_delete=models.CASCADE)
+    email = models.EmailField()
```

Comment:
Don't store email in the DB.

Reply:
The pattern we have is to store all the arguments for a background job on a model, and then kick off the job from the model or from an enqueuer. To remove this from the DB, we'd have to kick off the job directly from the serializer, when we have the email in-memory. Is that a trade-off we want to make? @davisagli I'd be interested in your input on this, especially.

Reply:
@wlonk @jgerigmeyer It's problematic to queue a job in rq before the models that are needed to run the job have been committed in a postgres transaction. I think the best path here might be:
1. keep the email field on the ScratchOrg model, with null=True
2. make sure it's not visible in the admin
3. make sure the scratch org creation job sets the field to null once it's no longer needed